src/functionalTest/resources/application.yaml

```diff
@@ -66,6 +66,9 @@ feature-toggle:
 xui:
   url: ${XUI_URL:https://manage-case.aat.platform.hmcts.net/cases/case-details}
 
+citizen:
+  dashboardUrl: ${CITIZEN_URL:https://wwww.example.com}s
```

Comment:
letter 's' at the end of URL is added by mistake or its required?

Reply:
Removed in next build